pages/search/search.jsx

```diff
@@ -81,22 +99,19 @@ class Search extends React.Component {
   }
 
   renderSearchBar() {
-    let classnames = classNames({activated: true, 'search-bar': true, 'w-100': true, 'mb-0': this.props.moderation});
-    let label;
-
-    if (this.props.moderation) {
-      label = <div className="mr-2">Keywords:</div>;
-    }
+    let classnames = classNames(`activated search-bar w-100`, {
+      'mb-0': this.props.moderation
+    });
 
     return <div className="d-flex align-items-center">
-            {label}
             <div className={classnames}>
               <DebounceInput id="search-box"
                               value={this.state.keywordSearched}
                               debounceTimeout={300}
                               type="search"
                               onChange={(event) => this.handleInputChange(event)}
-                              placeholder="Search keywords, people, tags..." />
+                              placeholder="Search keywords, people, tags..."
+                              className="form-control" />
```

Comment:
worth treating the `/>` the same as you would a `}`, so that you don't have to move around the `/>` when you add/remove properties:
```
<DebounceInput
  id="search-box"
  value={this.state.keywordSearched}
  debounceTimeout={300}
  type="search"
  onChange={(event) => this.handleInputChange(event)}
  placeholder="Search keywords, people, tags..."
  className="form-control"
/>
```